hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/s3/security/TestS3GetSecretRequest.java

```diff
@@ -378,7 +377,7 @@ public void testGetSecretWithTenant() throws IOException {
         new OMTenantAssignUserAccessIdRequest(
             new OMTenantAssignUserAccessIdRequest(
                 assignUserToTenantRequest(TENANT_ID,
-                    USER_BOB_SHORT, ACCESS_ID_BOB)
```

Comment:
We should revert the "bob" changes here and keep the "alice" changes. Because "bob" is used in tenant accessId testing while "alice" is for legacy s3v accessId.

Reply:
e.g. when we run tenant assign user CLI ([doc](https://ci-hadoop.apache.org/view/Hadoop%20Ozone/job/ozone-doc-master/lastSuccessfulBuild/artifact/hadoop-hdds/docs/public/feature/s3-tenant-commands.html#:~:text=Assign%20a%20user%20to%20a%20tenant)):

```bash
ozone tenant user assign testuser --tenant=tenantone
```

`testuser` is the `userPrincipal` in [handler](https://github.com/apache/ozone/blob/f0b30130911f4d7bacbe711ce51be798f8d8ba80/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/tenant/TenantAssignUserAccessIdHandler.java#L70), and is passed to OM as [`username`](https://github.com/apache/ozone/blob/f3dc1580c5dfd56634120c0863f5c297ec78842e/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java#L1051-L1056).